src/vstest.console/Processors/Utilities/StringUtilities.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.CommandLine.Processors.Utilities
+{
+    using System.Collections.Generic;
+    using System.Text;
+
+    internal static class StringExtensions
+    {
+        public static IEnumerable<string> Tokenize(this string input, char separator, char escape)
+        {
+            if (input == null) yield break;
```

Comment:
nit: logic should be in separate line within `{ }` for conditionals. R# with stylecop extension helps catch some of these.